systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainST.java

```diff
@@ -69,6 +71,86 @@ void testProducerConsumer() {
         ClientUtils.waitForClientSuccess(OAUTH_CONSUMER_NAME, NAMESPACE, MESSAGE_COUNT);
     }
 
+    @Test
+    void testProducerConsumerAudienceTokenChecks() {
+        LOGGER.info("Update Kafka broker configuration to use OAuth2 access token audience checks");
+        KafkaResource.replaceKafkaResource(oauthClusterName, kafka -> {
+            kafka.getSpec().getKafka().setListeners(new ArrayOrObjectKafkaListenersBuilder()
+                    .addNewGenericKafkaListener()
+                    .withName(Constants.PLAIN_LISTENER_DEFAULT_NAME)
+                    .withPort(9092)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(false)
+                    .withNewKafkaListenerAuthenticationOAuth()
+                    .withValidIssuerUri(keycloakInstance.getValidIssuerUri())
+                    .withJwksExpirySeconds(keycloakInstance.getJwksExpireSeconds())
+                    .withJwksRefreshSeconds(keycloakInstance.getJwksRefreshSeconds())
+                    .withJwksEndpointUri(keycloakInstance.getJwksEndpointUri())
+                    .withUserNameClaim(keycloakInstance.getUserNameClaim())
+                    .withEnablePlain(true)
+                    .withCheckAudience(true)
+                    .withClientId("kafka-component")
+                    .withTokenEndpointUri(keycloakInstance.getOauthTokenEndpointUri())
+                    .endKafkaListenerAuthenticationOAuth()
+                    .endGenericKafkaListener().build());
+        });
```

Comment:
Can you please do some indentation?

Reply:
There's not much to indent TBH, just KafkaListener. But sure. Already done.

Reply:
well listener is one part, another indent can be the Oauth part and `build` on newline etc.